telegram/user.py

```diff
@@ -99,3 +99,22 @@ def get_profile_photos(self, *args, **kwargs):
         Shortcut for ``bot.getUserProfilePhotos(update.message.from_user.id, *args, **kwargs)``
         """
         return self.bot.getUserProfilePhotos(self.id, *args, **kwargs)
+
+    @staticmethod
+    def de_list(data, bot):
```

Comment:
Just saying (don't think anything should be changed atm):
JSON can also be a list, so our notion of `de_json` & `to_json` should be fixed. (future todo?)